src/forges/GitLab/branches.jl

```diff
@@ -6,9 +6,39 @@
     developers_can_push::Bool
     developers_can_merge::Bool
     can_push::Bool
+    web_url::String
     commit::Commit
 end
 
 endpoint(::GitLabAPI, ::typeof(get_branch), owner::AStr, repo::AStr, branch::AStr) =
     Endpoint(:GET, "/projects/$(encode(owner, repo))/repository/branches/$branch")
 into(::GitLabAPI, ::typeof(get_branch)) = Branch
+
+
+# Get Repository Branches
+# https://docs.gitlab.com/ee/api/branches.html#list-repository-branches
+function endpoint(
+    ::GitLabAPI,
+    ::typeof(get_branches),
+    owner::AStr,
+    repo::AStr,
+)
+    return Endpoint(:GET, "/projects/$(encode(owner, repo))/repository/branches")
+end
+into(::GitLabAPI, ::typeof(get_branches)) = Vector{Branch}
+
+# Delete Branch
+# https://docs.gitlab.com/ee/api/branches.html#delete-repository-branch
+function endpoint(
+    ::GitLabAPI,
+    ::typeof(delete_branch),
+    owner::AStr,
+    repo::AStr,
+    branch_name::String,
+)
+    return Endpoint(
+        :DELETE,
+        "/projects/$(encode(owner, repo))/repository/branches/$(encode(branch_name))",
+    )
+end
+info(::GitLabAPI, ::typeof(delete_branch)) = Branch
```

Comment:
```suggestion
postprocessor(::GitLabAPI, ::typeof(delete_branch)) = DoNothing
```

If there's nothing returned, we indicate that with the `postprocessor` (which by default parses JSON)

Reply:
ohh interesting thanks!